src/workflow-engine/charms/temporal-worker/oci_factory/activities/activity_consume_events.py

```diff
@@ -52,6 +71,10 @@ async def consume(topic: str, consumer_group: str) -> dict:
             consumer.close()
 
     logging.info("Release: {}".format(value["release"]))
+    print("Release: {}".format(value["release"]))
```

Comment:
why a print here ?

Reply:
To have the temporal sdk catch the stdout.